tests/test_cmdline.py

```diff
@@ -164,8 +164,22 @@ async def print_sum(x, y):
 
 class TestCommandLineBasic(CmdlineTmpl):
     def test_no_file(self):
-        result = self.template(["python", "-m", "viztracer"], expected_output_file=None)
-        self.assertIn("help", result.stdout.decode("utf8"))
+        result = self.template(["python", "-m", "viztracer"], expected_output_file=None).stdout.decode("utf8")
```

Comment:
``result`` is used in ``template`` function and in this test file multiple times representing a ``subprocess``. I tend to not use the name for a different kind of object.

I think a better way here is to keep the ``result`` as it was before, and do a 

```
result_stdout = result.stdout.decode("utf-8")
```

You could potentially chain it and skipping the middle variable as well, I just think it would be clearer that way.